engine/src/main/java/org/terasology/logic/players/CameraClientSystem.java

```diff
@@ -76,4 +96,11 @@ public void resetCamera(ResetCameraEvent resetCameraEvent, EntityRef client, Aut
         }
         Location.attachChild(targetEntityForCamera, clientComponent.camera, cameraPosition, new Quat4f(Quat4f.IDENTITY));
```

Comment:
I do see your point of reducing the lines of code.  Thank you for your perspective.

The camera entity gets created from a prefab.  Theoretically a module could remove the location component from the prefab using a delta. That is how there is no guarantee.